app/scripts/lib/country-telephone-info.js

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ /**
+  * Country->phone number info.
+  */
+ define((require, exports, module) => {
+   'use strict';
+
+   module.exports = {
+     UK: {
```

Comment:
Fwiw, in the auth server I opted for `GB` instead of `UK` because it's the official ISO-3166-1 country code. Do you think it's worth using the same here?